guides/common/modules/ref_example-playbooks-based-on-modules-from-project-ansible-collections.adoc

```diff
@@ -0,0 +1,57 @@
+[id="example-playbooks-based-on-modules-from-{project-context}-ansible-collections"]
+= Example playbooks based on modules from {Project} Ansible Collections
+
+The following example playbooks are executed on `localhost`, which can also be {ProjectServer} itself.
+However, because Ansible modules from the {Project} Ansible Collections perform actions by using the {Project} API, you can run your playbooks on any system that can reach the {Project} API.
+
+All playbooks based on modules from {Project} Ansible Collections must include parameters detailing how to connect to the {Project} API.
+For more information, see xref:creating-a-playbook-with-modules-from-{project-context}-ansible-collections[].
+
+.Example Ansible playbook: Ensure that domain `new.example.com` exists in {Project}
+====
+The `{ansible-namespace}.domain` module can create, update, and delete domains.
+This example playbook uses `{ansible-namespace}.domain` to ensure that a domain named `new.example.com` exists and is managed by {Project}.
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+- name: Domain management
+  hosts: localhost
+  tasks:
+    - name: Ensure domain new.example.com exists
+      {ansible-namespace}.domain:
+        name: new.example.com
```

Comment:
this should reference vars files and the module defaults, as otherwise this example does not copy&paste to a working playbook.

Reply:
Added